### PR TITLE
01.01.01; 01.02.01; 01.03.01; 01.04.01; 01.05.01

### LINKED ISSUE #4: US 01.01.01 - Add habit
As a doer, I want to add a habit, giving it a brief title, reason, and date to start, and whether it is public or private..

### LINKED ISSUE #5: US 01.02.01 - Habit days of the week
As a doer, I want a habit to have a plan for what days of the week it should regularly occur.

### PR DESCRIPTION
**Pull request by marty-rudolf**
Add Habit functionality implemented. Testing and fixes to come.

Close #4 , Close #5

### CODE REVIEW

File: CMPUT301Project/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (schen32) [inline CMPUT301Project/.idea/.gitignore:1]: Remove this file. The .idea/ folder should not be included.

File: CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java [hunk 2]
```diff
@@ -47,4 +62,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // Access a Cloud Firestore instance from your Activity
         db = FirebaseFirestore.getInstance();
     }
+
+    public static FirebaseFirestore getFirestoreInstance(){
```
Comment (schen32) [inline CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java:66]: Seems redundant, just call FirebaseFirestore.getinstance directly. Also doesn't look like you're using this function anywhere.

**Comment by marty-rudolf** [comment]
Close #11 Close #7 Close #6

**Review by schen32 — APPROVED** [review]
Looks good.

**Pull request merged**